README.md

```diff
@@ -17,6 +17,39 @@ fastlane add_plugin cordova
 
 Then you can integrate it into your Fastlane setup:
 
+## Beta version
+```ruby
+platform :ios do
+  desc "Deploy new beta build"
+  lane :beta do
+    cert
+    sigh
+    gym(
+       scheme: "WheelChair88"
+    )
+    testflight
+    cordova(platform: 'ios') 
+    appstore(ipa: ENV['CORDOVA_IOS_RELEASE_BUILD_PATH'])
+  end
+end
+
+platform :android do
+  desc "Submit a new Beta Build"
+  lane :beta do
+    #gradle(task: "assembleRelease")
+     cordova(
+      platform: 'android',
+      keystore_path: './config/wheelchair88.keystore',
+      keystore_alias: 'wheelchair88',
+      keystore_password: 'Hopeti11end',
+      team_id:'KH5K2MDBCK'
+    )
+    # Cordova Fastlane Plugin
+    supply(apk: ENV['CORDOVA_ANDROID_RELEASE_BUILD_PATH'])
+  end 
+end
+```
+## Alpha version
```

Comment:
I feel like the lanes below were for actual production deployment, not alpha deployment, right? :\